tests/test_webhook.py

```diff
@@ -157,14 +159,15 @@ def test_extract_browser_label(self):
             actual = helpers.extract_browser_label(metadata_dict)
             self.assertEqual(expected, actual)
 
-    def test_extract_extra_label(self):
+    def test_extract_extra_labels(self):
         """Extract 'extra' label."""
         metadata_tests = [
-            ({'extra_label': 'type-media'}, 'type-media'),
+            ({'extra_labels': 'type-media'}, ['type-media']),
+            ({'extra_labels': 'cool, dude'}, ['cool', 'dude']),
```

Comment:
in the function you seem to expect unicode data (aka you do `encode('utf-8')`) so it's probably a good idea to make the test even with what is expected. and even create a test for a funky unicode string.

Reply:
Ah I also realized here that 

* `'cool,dude'` returns `['cool,dude']`
* `'cool , dude'` returns `['cool ', 'dude']`
* `' cool , dude'` returns `[' cool ', 'dude']`
* `', cool , dude'` returns `['', ' cool ', 'dude']`

just fyi